lib/Backend/Commands.hs

```diff
@@ -517,26 +518,26 @@ getEntryOrDir backend path =
       where
         go :: Path -> StateT Directory (Sem r) ()
         go rootPath = do
-          secrets <- lift $ fromMaybe [] <$> listSecrets backend rootPath
+          nodes <- lift $ fromMaybe [] <$> listDirectoryContents backend rootPath
           -- TODO: run in parallel
-          forM_ secrets \secret -> do
-            -- We need to find out whether `secret` is a directory name or an entry name.
+          forM_ nodes \node -> do
+            -- We need to find out whether `node` is a directory name or an entry name.
             --
             -- Here, we rely on the fact that Vault returns directory names suffixed by `/`.
             -- and entry names without any suffix.
             --
-            -- If `mkPathSegment` succeeds, then `secret` was not suffixed by `/`,
+            -- If `mkPathSegment` succeeds, then `entry` was not suffixed by `/`,
```

Comment:
```suggestion
            -- If `mkPathSegment` succeeds, then `node` was not suffixed by `/`,
```

Reply:
Done. I should be more attentive when making quick changes :)